### ISSUE TITLE
qml: too many buttons labelled "Pay". unclear which action is final

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#9276]
For on-chain payments at least, in the qml gui, I think the flow is unclear regarding which dialog is the "final" dialog: which button is the final button that if you click there is no going back.

---

Upon entering a bitcoin address, there is this first dialog. What happens if I click "Pay"? Does it send? But wait how do I set the fee, etc?
<details>
 <summary>pic1</summary>

![pic1](https://github.com/user-attachments/assets/0a65cc54-76fa-4c4b-acb9-320948149fe6)
</details>

---

Well, ok, I'll click it. Oh ok, I can tweak settings now. There is a "Pay" button again. What happens if I click it? Does it send?

<details>
 <summary>pic2</summary>

![pic2](https://github.com/user-attachments/assets/13c91df7-e723-4cad-9270-290577fdea96)
</details>

---

And actually in current releases, whether it sends depends on having a PIN code configured.
Without a PIN code, it signs and broadcasts directly. With a PIN code set, it prompts for it now.

(I've changed that in https://github.com/spesmilo/electrum/commit/c90c87afb951e0ef9099621d11f2a470a17f1972 to also show a final confirmation popup for the no-PIN case)

<details>
 <summary>pic3</summary>

![pic3](https://github.com/user-attachments/assets/90b4c285-302a-490f-a0da-f9965a88b4b5)
</details>

---

I am not sure what would be the best way to change this, but I think it should be clear to the user if pressing a button actually sends the tx. (I mean this in general, so also e.g. for the lightning flow)

Relatedly, the desktop GUI has a button labelled "Pay..." to suggest there will be a follow-up dialog: https://github.com/spesmilo/electrum/pull/6698

**Comment by accumulator** [comment]
agreed, noticed this too. I adapted the obvious ones on master, but for consistency we should also replace other non-final actions in dialogs, e.g. 'Open Channel', 'Close Channel' etc
